float64[] state
float64[] action
---
float64[] next_state
float32 node_probability